docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2133 @@
+
+Default: `"normal"`
+
+|`underline`
+|`boolean`
+|Underline the text? 
+
+Default: `false`
+
+|`italic`
+|`boolean`
+|Italicize the text? 
+
+Default: `false`
+
+|`align`
+|`string`
+|The horizontal alignment of text 
+
+Default: `left`
+|===
+
+*Returns:* `style`
+
+
+[float]
+[[formatdate_fn]]
+=== `formatdate`
+
+Format a valid date string or number of milliseconds since epoch using https://momentjs.com/[MomentJS]
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `format`
+|`string`
+|MomentJS Format with which to bucket, see the https://momentjs.com/docs/#/displaying/[MomentJS documentation]
+|===
+
+*Returns:* `string`
+
+
+[float]
+[[formatnumber_fn]]
+=== `formatnumber`
+
+Turns a `number` into a `string` using a NumeralJS format.
+
+*Accepts:* `number`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `format`
+|`string`
+|NumeralJS format string, see the http://numeraljs.com/#format[NumeralJS documentation]
+|===
+
+*Returns:* `string`
+
+
+[float]
+[[g_fns]]
+== G
+[[getCell_fn]]
+=== `getCell`
+
+Fetches a single cell in a table.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ 
+
+Aliases: `column`, `c`
+|`string`
+|The name of the column value to fetch, defaults to the first column
+
+|`row` 
+
+Alias: `r`
+|`number`
+|The row number, starting at 0 
+
+Default: `0`
+|===
+
+*Returns:* Depends on the data in the cell
+
+
+[float]
+[[gt_fn]]
+=== `gt`
+
+Returns whether the _context_ is greater than the argument.
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `value`
+|`number`, `string`
+|The value to compare the _context_ to
+|===
+
+*Returns:* `boolean`
+
+
+[float]
+[[gte_fn]]
+=== `gte`
+
+Returns whether the _context_ is greater than or equal to the argument.
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `value`
+|`number`, `string`
+|The value to compare the _context_ to
+|===
+
+*Returns:* `boolean`
+
+
+[float]
+[[h_fns]]
+== H
+[[head_fn]]
+=== `head`
+
+Gets the first N rows from the `datatable`. See also <<tail_fn>>.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ 
+
+Alias: `count`
+|`number`
+|Return this number of rows from the beginning of the `datatable` 
+
+Default: `1`
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[i_fns]]
+== I
+[[if_fn]]
+=== `if`
+
+Performs conditional logic.
+
+*Accepts:* `any`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ 
+
+Alias: `condition`
+|`boolean`
+|A `true` or `false`, usually returned by a subexpression. 
+If this is not supplied, then the input _context_ will be used.
+
+|`then`
+|`any`
+|The return value if `true`
+
+|`else`
+|`any`
+|The return value if `false`. If else is not specified, and the condition is `false`,
```

Comment:
For the second part, how about:

`When unspecified, and the condition is `false`, the input _context_ to the function is returned.